lib/widgets/sliders.dart

```diff
@@ -1,7 +1,9 @@
 import 'package:flutter/material.dart';
 import 'package:bars_frontend/main.dart';
 
-Widget getSliderInputRow(MyHomePageState homepageState, State context, MapEntry<String, dynamic> feature, double inputValue) {
+
+/// [context] is the widget that the input widget is on, so the widget that has to reload.
```

Comment:
```suggestion
/// [context] is the widget that the input widget is on, (i.e., the widget that has to rebuild on state change).
```

Do we want to say this three times?